Assets/MixedRealityToolkit/_Core/Interfaces/SpatialAwareness/IMixedRealitySpatialAwarenessSystem.cs

```diff
@@ -106,17 +104,10 @@ public interface IMixedRealitySpatialAwarenessSystem : IMixedRealityEventSystem,
         /// </summary>
         Material MeshOcclusionMaterial { get; set; }
 
-        /// <summary>
-        /// Gets the collection of <see cref="IMixedRealitySpatialAwarenessMeshDescription"/>s being tracked by the spatial awareness mesh subsystem.
-        /// </summary>
-        /// <returns>Dictionary of <see cref="IMixedRealitySpatialAwarenessMeshDescription"/>s, indexed by the mesh id.</returns>
-        Dictionary<uint, IMixedRealitySpatialAwarenessMeshDescription> GetMeshes();
-
         /// <summary>
         /// Gets the collection of <see cref="GameObject"/>s being managed by the spatial awareness mesh subsystem.
         /// </summary>
-        /// <returns>Dictionary of <see cref="GameObject"/>s, indexed by the mesh id.</returns>
-        Dictionary<uint, GameObject> GetMeshObjects();
+        Dictionary<int, GameObject> MeshObjects { get; }
```

Comment:
what prompted the change from `uint` to `int` type?

Reply:
+1

Reply:
Also, generally speaking I believe interface contracts should use interfaces rather than concrete types. `IDictionary<,>` over `Dictionary<,>` for example